src/Angor/Client/Pages/Create.razor

```diff
@@ -427,6 +425,8 @@
             return;
         }
 
+        project.ProjectInfo.CreationTransactionId = signedTransaction.GetHash().ToString();
```

Comment:
You also need to handle the new state in line 266 when checking on loading the page

Reply:
This is the result from the relay, what state do we need to handle here?

Reply:
We should only navigate to the view page if creation transaction id is populated

Reply:
Gotcha ill fix it